utils/move_and_process.py

```diff
@@ -153,6 +154,17 @@ def get_mlt_duration(filepath):
     fps = float(m.group(1))
     return frames/fps
 
+def get_loudness(filepath):
+    cmd = ['bs1770gain', '--xml', '--truepeak', filepath]
+    output = subprocess.check_output(cmd, stderr=subprocess.DEVNULL)
```

Comment:
In fact, we should also smoothly balk here, since many people won't have `bs1170gain` installed.

You could very well just do a blind try-except around calling `this.get_loudness()` so we just never really depend on this. It's just an added feature if it works and is available. We'd start noticing it being broken by not actually getting this data into new files in the database.

What do you think?

If not, we should have a better way to install these things for people who don't have them.

Reply:
I'm not sure I agree that measuring loudness should be seen as optional, but what the heck, lets do this for now. :)